File: C/1006.c
#include <stdio.h>
 
int main() {
 
    double A, B, C;
    scanf("%lf", &A);
    scanf("%lf", &B);
    scanf("%lf", &C);

    double MEDIA = (A * 2 + B * 3 + C * 5) / 10;
    printf("MEDIA = %.1f\n", MEDIA);
 
    return 0;
}

File: Javascript/1006.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let A = Number(lines.shift());
let B = Number(lines.shift());
let C = Number(lines.shift());

let MEDIA = (A * 2 + B * 3 + C * 5) / 10;
console.log("MEDIA = " + MEDIA.toFixed(1));